.github/workflows/ci.yml

```diff
@@ -86,3 +86,22 @@ jobs:
         docker buildx build --platform linux/amd64 --platform linux/arm64 \
         -t build-container-test .
 
+    - name: Test Container Image
+      run: |
+        docker run --rm \
+          build-container-test \
+          scan \
+          --filter-suite .github/vet/policy.yml \
+          --cloud-mode true \
```

Comment:
These are `vet-action` params. Not really vet. You just need to set the following env vars and then run `auth verify`. Example:

https://github.com/safedep/vet/pull/421/files